.github/workflows/tests.yml

```diff
@@ -276,8 +276,7 @@ jobs:
         env:
           CHECKOUT_TIMEOUT: 10
         run: |
-          GLOBIGNORE="plugins/chat/*";
-          LOAD_PLUGINS=1 RAILS_ENABLE_TEST_LOG=1 RAILS_TEST_LOG_LEVEL=error bin/turbo_rspec --use-runtime-info --profile=50 --verbose --format documentation plugins/*/spec/system
+          LOAD_PLUGINS=1 RAILS_ENABLE_TEST_LOG=1 RAILS_TEST_LOG_LEVEL=error bin/turbo_rspec --exclude-pattern="plugins/chat/*" --use-runtime-info --profile=50 --verbose --format documentation plugins/*/spec/system
```

Comment:
Before: https://github.com/discourse/discourse/actions/runs/14585556119/job/40910329681

```
Finished in 13 minutes 33 seconds (files took 0 seconds to load)
1532 examples, 0 failures, 62 pending
```

After: https://github.com/discourse/discourse/actions/runs/14585714028/job/40910748366?pr=32390

```
Finished in 12 minutes 33 seconds (files took 0 seconds to load)
1532 examples, 0 failures, 62 pending
```

Number of examples stayed the same which is a good indication that I didn't break anything.